### PR TITLE
Lhs all ids

### PR DESCRIPTION
**Pull request by oracleofnj**

### CODE REVIEW

File: src/main/transform.ml [hunk 1]
```diff
@@ -63,11 +63,12 @@ let expand_expressions (imports, globals, functions, externs) =
     (* If expression is not sufficiently atomic, create a new variable
        to hold the expression; return the new expression and whatever
        new statements are necessary to create the new variable *)
-      LitInt(i) -> (LitInt(i), [])
```
Comment (Neitsch) [inline src/main/transform.ml:66]: Is this not needed?

**Review by Neitsch — APPROVED** [review]

**Comment by oracleofnj** [comment]
The whole point of the PR was to remove that line! That was the line that
said "don't bother transforming ints into IDs."

**Comment by oracleofnj** [comment]
There's a wildcard e -> .....
that handles everything not (a) illegal (b) already an ID

**Comment by Neitsch** [comment]
Ah, now I see it. Feel free to merge

**Pull request merged**